s3router.js

```diff
@@ -44,7 +44,7 @@ function S3Router(options) {
      */
     router.get('/sign', function(req, res) {
         var filename = uuid.v4() + "_" + req.query.objectName;
-        var mimeType = mime.lookup(filename);
+        var mimeType = req.query.contentType;
```

Comment:
Need to double-check, but without this line, we can probably drop the entire `mime` dependency.